### PR TITLE
[New] How to Change Your Email

### PR DESCRIPTION
**Pull request by hzoppetti**
initial draft

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 3d8f089aed268ca756e1b48bf3b0b7869a5fb647

https://deploy-preview-3073--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 0fd41135a521090d671db464d4028a5373a6105c

https://deploy-preview-3073--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/quick-answers/linode-platform/how-to-change-your-email/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+## From the Cloud Manager
+
+1.  Log in to the [Cloud Manager](http://cloud.linode.com).
+
+1.  Click on your username and choose **My Profile** from the drop down menu.
+
+1.  On the My Profile page, in the **Email** field, enter the email you would like to associate with your account.
+
+    ![Click My Profile from Username Drop Down Menu](how-to-change-email-my-profile.png "Click My Profile from Username Drop Down Menu")
```
Comment (andystevensname) [inline docs/quick-answers/linode-platform/how-to-change-your-email/index.md]: Can this image be moved up to step two?

File: docs/quick-answers/linode-platform/how-to-change-your-email/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+1.  Click the **Save** button.
+
+1.  Cloud Manager will tell you that the email address has been updated. You will also receive an email at this new address from Linode Support confirming this update.
+
+    ![Click to Save a New Email](how-to-change-email-save.png "Click to Save a New Email")
```
Comment (andystevensname) [inline docs/quick-answers/linode-platform/how-to-change-your-email/index.md]: I think this image should be moved up to step 3.

File: docs/quick-answers/linode-platform/how-to-change-your-email/index.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+        curl -H "Content-Type: application/json" \
+        -H "Authorization: Bearer $TOKEN" \
+        -X PUT -d '{
+        "email": "jsmith@example.com"
+        }
+        }' \
+        https://api.linode.com/v4/account
+
+1.  You will get the same full account output, this time with the updated email account:
```
Comment (andystevensname) [inline docs/quick-answers/linode-platform/how-to-change-your-email/index.md]: I think "get" is always weak. Maybe: "The output will be the mostly same as in the previous step, but the email field will be updated with your new email address."

File: docs/quick-answers/linode-platform/how-to-change-your-email/index.md (hunk 2 above)
Comment (hzoppetti) [reply docs/quick-answers/linode-platform/how-to-change-your-email/index.md]: I moved this to under the Save button

File: docs/quick-answers/linode-platform/how-to-change-your-email/index.md (hunk 1 above)
Comment (hzoppetti) [reply docs/quick-answers/linode-platform/how-to-change-your-email/index.md]: Moved

File: docs/quick-answers/linode-platform/how-to-change-your-email/index.md (hunk 3 above)
Comment (hzoppetti) [reply docs/quick-answers/linode-platform/how-to-change-your-email/index.md]: Updated

**Pull request merged**